microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -36,6 +40,14 @@ This release adds improvements related to the following components and concepts.
 ** If you run cAdvisor as a standalone DaemonSet for monitoring pods and containers, update it to v0.43.0 or later.
 ** If you deploy Java applications with the JDK, ensure you are using JDK 11.0.16 and later or JDK 15 and later, which fully support cgroup v2.
 
+[id="microshift-4-14-updates-supported"]
+=== Updates are supported on {product-version} and later
+With this release, both minor-version and patch releases updates are supported.
+
+* {product-title} offers in-place updates on {op-system-ostree} systems with automatic system rollback capabilities and automatic back up and restore functions.
+* Updates of the RPMs on a non-OSTree system such as Red Hat Enterprise Linux (RHEL) are also supported.
```

Comment:
Kind of a weird situation: does RHEL need to be spelled out if it's already mentioned as an abbreviation in the preceding list item?